typeDefs.js

```diff
@@ -104,6 +149,33 @@ const typeDefs = gql`
     getAllUsers: [User]
     getUserByToken: User
     getUserById(id: ID!): User
+
+    getAllProducts: [Products!]!
+    getProductsById(id: ID!): Products
+    getProductsByOptions(
+      filter: FilterInput
+      limit: Int
+      skip: Int
+      search: String
+      sort: SortInput
+    ): [Products]!
+
+    getAllComments: [Comments]
+    getCommentsById(id: ID!): Comments
```

Comment:
I don't see a definition of the query for the `getCommentsByProduct`

Reply:
added